superset-frontend/packages/generator-superset/package.json

```diff
@@ -2,40 +2,40 @@
   "name": "@superset-ui/generator-superset",
   "version": "0.18.25",
   "description": "Scaffolder for Superset",
+  "keywords": [
+    "yeoman",
+    "generator",
+    "superset",
+    "yeoman-generator"
+  ],
+  "homepage": "https://github.com/apache-superset/superset-ui#readme",
```

Comment:
While we're here, maybe we want to move this to reference the `apache/superset` repo?

Reply:
Yeah! Good Catch!